src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -363,4 +381,72 @@ public Collection<DataAccessRequest> getApprovedDARsForDataset(Dataset dataset)
     return dataAccessRequestDAO.findApprovedDARsByDatasetId(dataset.getDatasetId());
   }
 
+  public void sendExpirationNotices() {
+    sendDARExpirationReminderNotices();
+    sendDARExpirationNotices();
+  }
+
+  private void sendDARExpirationNotices() {
+    EmailType emailType = EmailType.DAR_EXPIRED;
+    sendDARMessageToList(emailType, EXPIRE_NOTICE_INTERVAL);
+  }
+
+  private void sendDARExpirationReminderNotices() {
+    EmailType emailType = EmailType.DAR_EXPIRATION_REMINDER;
+    sendDARMessageToList(emailType, EXPIRE_WARN_INTERVAL);
+  }
+
+  private void sendDARMessageToList(EmailType type, String interval) {
+    List<DataAccessRequest> expiredDars = dataAccessRequestDAO.findAgedDARsByEmailTypeOlderThanInterval(
+        type.getTypeInt(), interval, MINIMUM_SUBMITTED_DATE_FOR_DAR_EXPIRATIONS);
+    expiredDars.forEach(expiredDar -> {
+      try {
+        String referenceId = expiredDar.getReferenceId();
+        User user = userDAO.findUserById(expiredDar.getUserId());
+        String darCode = expiredDar.getDarCode();
+        String userName = user.getDisplayName();
+        if (user.getEmail() == null) {
+          // Do not throw here.  Log information about the DAR since this will continue
+          // to appear broken until manual intervention is taken to resolve the missing user
+          // email address
+          logException(new Exception(String.format(
+              "Email address for user %d (%s) not found for expiring warning.  DAR reference id: %s",
+              expiredDar.getUserId(), userName, referenceId)));
```

Comment:
Since this is inside a try/catch that logs the exception, you could throw here and it would be caught and logged. That way you don't need to use `else`
```suggestion
          throw new Exception(String.format(
              "Email address for user %d (%s) not found for expiring warning.  DAR reference id: %s",
              expiredDar.getUserId(), userName, referenceId));
```